.github/workflows/release.yaml

```diff
@@ -83,6 +83,8 @@ jobs:
     needs: [wait-for-checks]
     runs-on: macos-latest # Due to our code signing process, it's vital that we run our release steps on macOS.
     steps:
+      - uses: docker-practice/actions-setup-docker@master
```

Comment:
It'd be good to use a tag or commit if available — we've been hit in the past with GH actions changing out from under us

Reply:
updated to v1